src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallService {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    CinemaHall get(Long id);
```

Comment:
The `get` method should return an `Optional<CinemaHall>` instead of `CinemaHall` to handle cases where the cinema hall might not be found, aligning with the DAO layer's return type.